sale_order_product_recommendation/wizards/sale_order_recommendation.py

```diff
@@ -270,12 +265,12 @@ def _get_last_sale_price_product(self):
                 [
                     ("company_id", "=", self.env.user.company_id.id),
                     ("partner_id", "=", self.partner_id.id),
-                    ("confirmation_date", "!=", False),
+                    ("date_order", "!=", False),
```

Comment:
This condition should be changed to test the state instead, as now `date_order` is always going to be filled.

Reply:
Condition's been changed to test the state.